examples/service-clients/odsp-client/shared-tree-demo/webpack.config.cjs

```diff
@@ -61,6 +61,12 @@ module.exports = (env) => {
 					title: "Hello Demo",
 					favicon: "",
 				}),
+				new webpack.DefinePlugin({
```

Comment:
In the `FluidExamples` repo, we use the `dotenv-webpack` plugin to accomplish this. Can we not use that here?
https://github.com/microsoft/FluidExamples/blob/main/item-counter-spe/webpack.config.cjs#L64

Reply:
Fixed